Gruntfile.js

```diff
@@ -8,6 +8,11 @@ module.exports = function (grunt) {
   var pkg = grunt.file.readJSON('package.json');
 
   grunt.initConfig({
+    mocha: {
```

Comment:
Instead of embedding `mocha` here, can you create a new file in the `tasks` subdirectory called mocha.js that has this configuration?